spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/BaseProcedure.java

```diff
@@ -160,6 +163,17 @@ protected void refreshSparkCache(Identifier ident, Table table) {
     cacheManager.recacheByPlan(spark, relation);
   }
 
+  protected Expression filterExpression(Identifier ident, String where) {
+    try {
+      String name = Spark3Util.quotedFullIdentifier(tableCatalog().name(), ident);
```

Comment:
Just use `tableCatalog` and `spark` for consistency with other methods since this class holds the vars?

Reply:
Good catch